Pointwise.lua

```diff
@@ -31,11 +31,14 @@ function Pointwise:createIODescriptors(input)
    end
 
    local nElem = input:nElement()
-   self.nElem = self.nElem or nElem -- this goes to the second branch only once
-   if self.iDesc and nElem == self.nElem then return end
-   self.nElem = nElem
-   self.iDesc = cudnn.toDescriptor(input:view(1,1,1,nElem))
-
+   self.nElem = self.nElem or nElem
+   if self.iDesc and nElem == self.nElem and
```

Comment:
No, it doesn't hold a pointer to the data.... You can see it also by the signature of cudnnSetTensorNdDescriptor, as called within init.lua for cudnn.toDescriptor(t): 
```
   errcheck('cudnnSetTensorNdDescriptor', descriptor[0], cudnnDataType,
            t:dim(), size:data(), stride:data())
```